### PR TITLE
Document how to make a release

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: docs/make_a_release.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+This documents how a core developer should create a release.
+
+0. Configure `~/.pypirc` as described
+   `here <http://peterdowns.com/posts/first-time-with-pypi.html>`_.
+
+1. Make sure that all CI tests are passing.
+
+2. Create the source tarball and binary wheels::
+
+   git checkout master
```
Comment (jcfr) [inline docs/make_a_release.rst]: git fetch upstream

**Comment by jcfr** [comment]
One nitpick and we should be all set. Yeahh

File: docs/make_a_release.rst (hunk 1 above)
Comment (thewtex) [inline docs/make_a_release.rst]: Yes! Done

**Comment by jcfr** [comment]
Thanks :+1:

**Pull request merged**